js/focuspoint/jquery.focusPoint.js

```diff
@@ -408,8 +408,7 @@
 	{
 		return this.each(function ()
 		{
-			(new $.imageFocus.focusPoint(this,
-				options));
+			(new $.imageFocus.focusPoint(this, options)); // eslint-disable-line no-new
```

Comment:
We're gonna change this after the meetup?
Anyways. We shouldn't disable checks. Because then we'll loose sight of them. Better to leave them be than to hide them.